api_tests/e2e/rfc003/btc_eth/resume.ts

```diff
@@ -0,0 +1,112 @@
+import { sleep } from "../../../lib/util";
```

Comment:
I focused my review on the rest of the code, so I haven't looked at this yet. I hope that's okay.

Reply:
Any review is good review - thanks.